connectd/connectd_wire.csv

```diff
@@ -59,6 +59,7 @@ msgdata,connectd_connect_failed,addrhint,?wireaddr_internal,
 msgtype,connectd_peer_connected,2002
 msgdata,connectd_peer_connected,id,node_id,
 msgdata,connectd_peer_connected,addr,wireaddr_internal,
+msgdata,connectd_peer_connected,remote_addr,wireaddr_internal,
```

Comment:
Make this optional?  You can use `wireaddr_internal?`

Reply:
Actually, make that `wireaddr?` since this isn't an internal thing.

Reply:
Yes, I will change it to wireaddr. Already had that in mind